xml/dm_registers.xml

```diff
@@ -153,8 +148,8 @@
             Debug Module after power up, including the platform's system reset
             or Debug Transport reset signals.
 
-            A debugger should pulse this bit low to ensure that the Debug
-            Module is fully reset and ready to use.
+            When first connecting, a debugger should pulse this bit low to
```

Comment:
Right, but it doesn't need to be in the *spec* that debugger SHOULD do this (in the IEEE meaning of SHOULD). It's just something the debugger is allowed to do.